### DISCUSSION TITLE
Raise the honor cap

### DISCUSSION THREAD

**Discussion by finurlig** [chromiecraft/chromiecraft#4003] [Suggestions]
Farming the marks for the newly released PvP gear gives you a large amount of honor. That will probably cap your honor before you farmed the marks you need for a piece of gear and . Raising the honor cap would make it a lot more convenient for a grind that is already long and tedious. My suggestion is to raise it to 150k instead of only 75k.

**Comment by Amrx2** [comment]
Hi finurlig, we had it previously at 150k before we dropped it to the current blizzlike cap of 75k so people don't store honor so they can get all items immediately.

As well as the mounts you can spend on honor, you can also trade it to gold as well to earn some money if you have extra.
